swift/ql/lib/codeql/swift/dataflow/internal/TaintTrackingPrivate.qll

```diff
@@ -36,11 +36,32 @@ private module Cached {
       nodeTo.asDefinition().(Ssa::WriteDefinition).isInoutDef(e)
     )
     or
+    // allow flow through `try!` and similar constructs
+    // TODO: this should probably be part of DataFlow / TaintTracking?
+    nodeFrom.asExpr() = nodeTo.asExpr().(AnyTryExpr).getSubExpr()
+    or
+    // allow flow through `!`
+    // TODO: this should probably be part of DataFlow / TaintTracking?
+    nodeFrom.asExpr() = nodeTo.asExpr().(ForceValueExpr).getSubExpr()
```

Comment:
Should these be in `DataFlow` rather than `TaintTracking`?

Reply:
Yes, do you know where in `DataFlow` they should be?

Reply:
It should be in this predicate: https://github.com/github/codeql/blob/main/swift/ql/lib/codeql/swift/dataflow/internal/DataFlowPrivate.qll#L87

Reply:
Thanks!

Reply:
Done.